MentorBook.Data/Repositories/UserMockingRepository.cs

```diff
@@ -45,5 +46,10 @@ public List<User> GetUserByFilter(string fillteringVlaue)
             return _allUsersInTheUniverse;
         }
 
+        public UserAdditionalInfoModel GetUserAdditionalInfoByUserId(int userId)
+        {
+            return _allUsers.FirstOrDefault(uid => uid.UserId == userId);
```

Comment:
Why single value? So let's say in the database we keep records for his high school and university, why are we going to show only one of them?